### ISSUE TITLE
Edge not accepting name change

### ISSUE THREAD

**Issue by jjsenay** [ambianic/ambianic-edge#388]
MAC OS
BRAVE BROWSER

All, went an updated my UI PWA client, via refresh.  Go the new feature to name the Edge.  Selected a new name.  Tried to sync with the Edge.  Got this error message:

Error updating display name on edge device. Could be offline or outdated.

Notice the Edge is 1.17.0  SSH into Edge.  Ran` ambianic upgrade`

Restarted UI.PWA, attached via Remote Edge.

Downloaded upgrade and restarted on its own.
11:05
Went to the UI.PWA and tried to change my Edge name.  Got the same error.  Does not rename the Edge

JSConsole:

settings~e0bffda6.d62e6773.js?__WB_REVISION__=2a4a2ede3251a3d81d26:1 Exception calling setDeviceDisplayName() Error: PeerFetch Timeout while waiting for response.
    at n.<anonymous> (app~882e284f.72e07aa6.js?__WB_REVISION__=d5caa9df4177c672ca7f:1)
    at x (vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1)
    at Generator._invoke (vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1)
    at Generator.next (vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1)
    at n (vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1)
    at i (vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1) Error: PeerFetch Timeout while waiting for response.
    at n.<anonymous> (https://ui.ambianic.ai/js/app~882e284f.72e07aa6.js?__WB_REVISION__=d5caa9df4177c672ca7f:1:28727)
    at x (https://ui.ambianic.ai/js/vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1:3901)
    at Generator._invoke (https://ui.ambianic.ai/js/vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1:4996)
    at Generator.next (https://ui.ambianic.ai/js/vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1:4081)
    at n (https://ui.ambianic.ai/js/vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1:3328)
    at i (https://ui.ambianic.ai/js/vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1:3531)
(anonymous) @ settings~e0bffda6.d62e6773.js?__WB_REVISION__=2a4a2ede3251a3d81d26:1
x @ vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1
(anonymous) @ vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1
(anonymous) @ vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1
n @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
a @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
Promise.then (async)
n @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
i @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
(anonymous) @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
(anonymous) @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
onDisplayNameChanged @ settings~e0bffda6.d62e6773.js?__WB_REVISION__=2a4a2ede3251a3d81d26:1
(anonymous) @ chunk-2d0e6c87.dea32d68.js?__WB_REVISION__=dab53e9457d9a2a354df:1
x @ vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1
(anonymous) @ vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1
(anonymous) @ vendors~ccfd30e1.dfa0311a.js?__WB_REVISION__=7a007d4bec7a4bb195cf:1
n @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
i @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
(anonymous) @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
(anonymous) @ vendors~192b16d6.1730cfaa.js?__WB_REVISION__=91366d4a1690248dad06:1
e @ chunk-2d0e6c87.dea32d68.js?__WB_REVISION__=dab53e9457d9a2a354df:1
re @ vendors~d2305125.d9f1572f.js?__WB_REVISION__=963be5e3f2b17ef960e0:7
n @ vendors~d2305125.d9f1572f.js?__WB_REVISION__=963be5e3f2b17ef960e0:7
(anonymous) @ vendors~d2305125.d9f1572f.js?__WB_REVISION__=963be5e3f2b17ef960e0:
[JSCONSOLE_Name_Edge.txt](https://github.com/ambianic/ambianic-edge/files/7282700/JSCONSOLE_Name_Edge.txt)
7

**Comment by ivelin** [comment]
Yes. That's because the required update for the edge software has not been released yet. Coming up next.
